test/debug/test_charptr.py

```diff
@@ -235,4 +243,4 @@ def clear_raw_data_in_closed_handles():
             assert h.raw_data_size == -1
     finally:
         _debug.set_protected_raw_data_max_size(old_raw_data_max_size)
-        _debug.set_closed_handles_queue_max_size(old_closed_handles_max_size)
\ No newline at end of file
+        _debug.set_closed_handles_queue_max_size(old_closed_handles_max_size)
```

Comment:
I am not sure what my editor did here: was an empty line missing?

Reply:
Yup.